test/models/user_test.rb

```diff
@@ -395,172 +394,171 @@
+        auth[:provider] = 'wrong_provider'
+        proc { User.find_from(auth) }
+          .must_raise(ActiveRecord::RecordNotFound)
       end
     end
 
-    context "auth[:uid]が異なるとき" do
-      it "例外が起きること" do
+    context 'auth[:uid]が異なるとき' do
+      it '例外が起きること' do
         auth = @auth.dup
-        auth[:uid] = "wrong_uid"
-        proc { User.find_from( auth ) }.
-          must_raise(ActiveRecord::RecordNotFound)
+        auth[:uid] = 'wrong_uid'
+        proc { User.find_from(auth) }
+          .must_raise(ActiveRecord::RecordNotFound)
       end
     end
 
-    context "auth[:provider]、auth[:uid]が同じとき" do
-      it "例外が起きること" do
-        proc { User.find_from( @auth ) }.
-          must_raise(ActiveRecord::RecordNotFound)
+    context 'auth[:provider]、auth[:uid]が同じとき' do
+      it '例外が起きること' do
+        proc { User.find_from(@auth) }
+          .must_raise(ActiveRecord::RecordNotFound)
       end
     end
   end
 
-  describe "#read_notices / #unread_notices" do
+  describe '#read_notices / #unread_notices' do
     before { @user = create(:user) }
 
-    context "きづきがないとき" do
-      it "#read_noticesが0件であること" do
+    context 'きづきがないとき' do
+      it '#read_noticesが0件であること' do
         @user.read_notices.count.must_equal 0
       end
 
-      it "#unread_noticesが0件であること" do
+      it '#unread_noticesが0件であること' do
         @user.unread_notices.count.must_equal 0
       end
     end
 
-    context "きづきが1件のとき" do
+    context 'きづきが1件のとき' do
       before { @notice = create(:notice) }
 
-      it "#read_noticesが0件であること" do
+      it '#read_noticesが0件であること' do
         @user.read_notices.count.must_equal 0
       end
 
-      it "#unread_noticesが1件であること" do
+      it '#unread_noticesが1件であること' do
         @user.unread_notices.count.must_equal 1
         @user.unread_notices.first.must_equal @notice
       end
     end
 
-    context "読まれたきづきが1件のとき" do
+    context '読まれたきづきが1件のとき' do
       before do
         @notice = create(:notice)
         @notice.read_by @user
       end
 
-      it "#read_noticesが1件であること" do
+      it '#read_noticesが1件であること' do
         @user.read_notices.count.must_equal 1
         @user.read_notices.first.must_equal @notice
       end
 
-      it "#unread_noticesが1件であること" do
+      it '#unread_noticesが1件であること' do
         @user.unread_notices.count.must_equal 0
       end
     end
 
-    context "ほかのひとに読まれたきづきが1件のとき" do
+    context 'ほかのひとに読まれたきづきが1件のとき' do
       before do
         @another_user = create(:user)
         @notice = create(:notice)
         @notice.read_by @another_user
       end
 
-      it "#read_noticesが1件であること" do
+      it '#read_noticesが1件であること' do
         @user.read_notices.count.must_equal 0
       end
 
-      it "#unread_noticesが1件であること" do
+      it '#unread_noticesが1件であること' do
         @user.unread_notices.count.must_equal 1
         @user.unread_notices.first.must_equal @notice
       end
     end
 
-    context "読んだきづきと読んでいないものが1件ずつあるとき" do
+    context '読んだきづきと読んでいないものが1件ずつあるとき' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.